### PR TITLE
[MRG+1] Document copying of spider arguments to attributes

### PR DESCRIPTION
**Pull request by Digenis**
I refer to https://github.com/scrapy/scrapy/blob/1.3/scrapy/spiders/__init__.py#L30
I'm too verbose and I need some feedback to start editing out.
Also, should we include something about calling `super()` in subclasses?

### CODE REVIEW

**Comment by kmike** [comment]
A great addition to the docs 👍 This is an useful feature which was barely documented.

**Comment by Digenis** [comment]
How about the way I document it?
Isn't it too verbose?

**Comment by kmike** [comment]
No, I think the verbosity level is just right.

File: docs/topics/spiders.rst [hunk 1]
```diff
@@ -297,6 +297,37 @@ Spiders can access arguments in their `__init__` methods::
...
+    class MySpider(scrapy.Spider):
+        name = 'myspider'
+
+        def start_requests(self):
+            yield scrapy.Request('http://www.example.com/categories/%s' % self.category)
+
+Keep in mind that spider arguments are only strings.
+The spider will not do any parsing on its own.
+If you were to set the `start_urls` attribute from the command line,
+you would have parse it on you own into a list
```
Comment (Digenis) [inline docs/topics/spiders.rst]: typo: s/would have parse it/would have to parse it/

File: docs/topics/spiders.rst [hunk 2]
```diff
@@ -297,6 +297,37 @@ Spiders can access arguments in their `__init__` methods::
...
+    class MySpider(scrapy.Spider):
+        name = 'myspider'
+
+        def start_requests(self):
+            yield scrapy.Request('http://www.example.com/categories/%s' % self.category)
+
+Keep in mind that spider arguments are only strings.
+The spider will not do any parsing on its own.
+If you were to set the `start_urls` attribute from the command line,
+you would have to parse it on you own into a list
```
Comment (voith) [inline docs/topics/spiders.rst]: on <strike>you</strike> `your` own

**Comment by redapple** [comment]
Thank you @Digenis (and @voith )

**Pull request merged**